pages/hello.py

```diff
@@ -40,6 +40,7 @@ def invalid_conversation_text(self):
 
     @property
     def is_header_title_visible(self):
+        self.wait_for_element_visible(*self._header_title_locator)
         return self.is_element_visible(*self._header_title_locator)
```

Comment:
Thanks for the pull request, @Standard8!  You could probably replace this second line with just `return True`, as the wait that you added will ensure that the element is visible before this line is reached.

Having said that, I'm not totally certain that this wait will be sufficient. Unfortunately WebDriver's logic for determining if an element is visible isn't always foolproof, so often we need to check something else (e.g., a class that is added, the element's position, etc.). I think for now this is a good enhancement, but if you find there are still intermittent failures then we might have to look for a better wait.

r+ wc (with comments)